src/components/WizardModal/WizardModal.jsx

```diff
@@ -80,18 +84,32 @@ class WizardModal extends Component {
   };
 
   handlePrevious = () => {
-    this.setState(state => ({ step: state.step - 1 }));
+    const stepBack = this.state.step - 1;
+    this.changedBack(stepBack);
+    this.setState(() => ({ step: stepBack }));
```

Comment:
Do we need to validate here if we've gone back? `handlePrevious`'s purpose is to decrement the `step`. I think we can just call the callback directly.

Also since we're using existing state to update the state and [there is no guarantee when setState is called](https://reactjs.org/docs/react-component.html#setstate), _I think_ we need to use the callback/updater function still.

```suggestion
    this.setState(
      state => ({ step: state.step - 1 }),
      () => {
        if (this.props.onBack) {
          this.props.onBack(this.state.step);
        }
      }
    );
```